src/labels/CustomLabels.labels

```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Video Duration displayed in the Video Header Component</shortDescription>
         <value>in 2 minutes</value>
     </labels>
+    <labels>
+        <fullName>gseuChecklistCreateConstituentRecordsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for Create Constituent Records section in Get Started</shortDescription>
+        <value>People are the heart of any nonprofit. Explore Nonprofit Success Pack features to learn how to manage your constituents and their household, personal, and professional connections.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistCreateConstituentRecordsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Create Constituent Records section in Gest Started</shortDescription>
+        <value>Create Constituent Records</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Blue button text for Connect Contacts to Organization checklist item</shortDescription>
```

Comment:
I don't think we want to use colors for accessibility reasons. I would just say: Button label for Connect Contacts to Organization checklist item